faculty_cli/templates.py

```diff
@@ -0,0 +1,61 @@
+"""Interact with the Faculty knowledge centre."""
+
+# Copyright 2016-2019 Faculty Science Limited
```

Comment:
```suggestion
# Copyright 2016-2020 Faculty Science Limited
```

Reply:
Should we make this change as a separate PR across the whole repo? Happy to do it FYI

Reply:
Dealt with here: https://github.com/facultyai/faculty-cli/pull/40